crates/swc_ecma_transforms_proposal/src/decorator_2022_03.rs

```diff
@@ -1699,7 +1699,8 @@ impl VisitMut for Decorator202203 {
         let old_extra_lets = self.extra_lets.take();
         let old_extra_vars = self.extra_vars.take();
 
-        let mut new = Vec::with_capacity(n.len());
```

Comment:
@kdy1 do you have any idea why this `n.len()` might sometimes be returning a very large `1314431791570856 / sizeof(Stmt)`? We noticed it only after bumping to the latest swc patches in only some scenarios https://github.com/denoland/deno/pull/23187#issuecomment-2036260579

Reply:
Cqn you try reverting https://github.com/dudykr/ddbase/pull/27 ?
You can clone it and add `[patch.crates.io]` to the local clone. cc @bgw

Reply:
Thanks! It was actually locked to 0.2.6. I'll try 0.2.8 and see.

Reply:
Same issue in 0.2.8 as in 0.2.6. I'll try harder tomorrow on getting a local reproduction so that I can isolate the problem. It only seems to happen on the CI atm for some reason.

Reply:
This is definitely a corruption bug -- do these two 32-bit numbers look familiar? Indices perhaps?

```
>>> hex(1314431791570856)
'0x4ab7800049fa8'
>>> [0x49fa8, 0x4ab78]
[303016, 306040]
```